clients/privacy-center/components/modals/consent-request-modal/ConsentRequestForm.tsx

```diff
@@ -97,6 +98,15 @@ const useConsentRequestForm = ({
           return;
         }
 
+        // After successfully initializing a consent request, save the current
+        // cookie with our unique fides_user_device_id, etc.
+        try {
+          setFidesCookie(cookie);
```

Comment:
wouldn't the cookie already be saved from `getOrCreateFidesCookie` at the top?

Reply:
Nope- as I mentioned in my mega comments above, that `getOrCreateFidesCookie` function has no side-effects so it's up to the caller to save the cookie.